apertium-kaa.kaa.lexc

```diff
@@ -8363,6 +8364,7 @@ alıstır:alıstır V-TV-CAUS ;
 alıs:alıs V-TV ;
 alıs:alıs V-IV ;
 al:al V-TV ;
+alıw:alıw V-IV-COOP ;
```

Comment:
This is a verbal noun form of al-, not a cooperative form.  If you want a cooperative form, you should categorise alıs- this way.